server/src/test/java/com/objectcomputing/checkins/services/checkin_notes/CheckinNoteControllerTest.java

```diff
@@ -13,15 +13,16 @@
  import io.micronaut.http.HttpStatus;		
  import io.micronaut.http.client.HttpClient;		
  import io.micronaut.http.client.annotation.Client;		
```

Comment:
There's something wonky going on with the whitespace in this file.

Reply:
there is, switched from vs code to IntelliJ so it messed it up.